global_helpers/panther_base_helpers.py

```diff
@@ -309,6 +310,60 @@ def deep_get(dictionary: dict, *keys, default=None):
     return out
 
 
+# pylint: disable=R1260,R0911
```

Comment:
```suggestion
# pylint: disable=too-complex,too-many-return-statements
```

Reply:
Addressed in [`37575e8` (#819)](https://github.com/panther-labs/panther-analysis/pull/819/commits/37575e805c2c8a2cb3b7ad02c92bc00bc693fa42).